yarn-project/archiver/src/archiver/kv_archiver_store/log_store.ts

```diff
@@ -60,7 +60,7 @@ export class LogStore {
             dataStartIndexForTx,
             block.number,
             /* isFromPublic */ false,
-            log.toBuffer(),
+            log.toFields(),
```

Comment:
TxScopedL2Log now represents the log as fields and not as a buffer. I did this change because Fields are the natural representation and it results in less conversions in the code.